pkgs/tools/audio/whisper-ctranslate2/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, fetchFromGitHub
+, nix-update-script
+, python3
+, python3Packages
+}:
+let
+  pname = "whisper-ctranslate2";
+  version = "0.2.7";
+in
+python3.pkgs.buildPythonApplication {
+  inherit pname version;
+  format = "setuptools";
+
+  disabled = python3.pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "Softcatala";
+    repo = pname;
+    rev = version;
+    hash = "sha256-dUmQNKgH+SIlLhUEiaEGXUHZQDr3fidsAU2vATJiXBU=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    numpy
+    faster-whisper
+    ctranslate2
+    tqdm
+    sounddevice
+    tqdm
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
+    description = "Whisper command line client compatible with original OpenAI client based on CTranslate2";
+    homepage = "https://github.com/Softcatala/whisper-ctranslate2";
+    changelog = "https://github.com/Softcatala/whisper-ctranslate2/releases/tag/${version}";
+    license = licenses.asl20;
```

```suggestion
    license = licenses.mit;
```
It looks like MIT License.
https://github.com/Softcatala/whisper-ctranslate2/blob/f24872f7885451aba982c5d42616aa0386e1884a/LICENSE#L1